# create configure file that does not fail on cross compile
# avoid check of HAVE_STRTONUM when cross compiling
# 
--- configure.ac.orig	2021-06-10 08:20:51.000000000 +0000
+++ configure.ac	2021-11-06 14:12:24.093353400 +0000
@@ -160,6 +160,7 @@
          [return (strtonum("0", 0, 1, NULL) == 0 ? 0 : 1);]
 	 )],
 	 [AC_DEFINE(HAVE_STRTONUM) AC_MSG_RESULT(yes)],
+	 [AC_LIBOBJ(strtonum) AC_MSG_RESULT(no)],
 	 [AC_LIBOBJ(strtonum) AC_MSG_RESULT(no)]
 )
 
